Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/MatchIdentifier.java

```diff
@@ -0,0 +1,75 @@
+package org.cqframework.cql.cql2elm.model;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MatchIdentifier {
+
+    private Match matchType;
+    private Object resolvedIdentifier;
```

Comment:
Is this the resolved _element_? If so it should be named as such, this makes it sound like it should be a string and it's the actual resolved identifier, not the element that was resolved.